plugin/clone/src/clone_hton.cc

```diff
@@ -105,6 +106,15 @@ static bool run_hton_clone_begin(THD *thd, plugin_ref plugin, void *arg) {
   return (false);
 }
 
+static bool run_collect_expected_storage_vector(THD *, plugin_ref plugin,
+                                                void *arg) {
+  auto clone_arg = static_cast<std::unordered_set<legacy_db_type> *>(arg);
+  auto hton = plugin_data<handlerton *>(plugin);
+  if (hton->db_type == DB_TYPE_INNODB || hton->db_type == DB_TYPE_ROCKSDB)
```

Comment:
We could collect clone-supporting plugins, i.e. whose handlertons have `clone_interface.clone_begin != nullptr`. This resolves the issue of other SEs and avoids hardcoding InnoDB/MyRocks here.